src/stories/components/Slider/Slider.js

```diff
@@ -0,0 +1,91 @@
+// Awesome Swiper React component credits: https://swiperjs.com/react
+
+import './styles.scss';
+import 'swiper/swiper.scss'; // Swiper styles
+
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { Swiper, SwiperSlide } from 'swiper/react';
+
+const TABLET_SCREEN_SIZE = 640;
+const DESKTOP_SCREEN_SIZE = 1024;
+const WIDESCREEN_SCREEN_SIZE = 1400;
+
+
+/**
+ * Component for branding element.
+ *
+ * @component
+ * @param {node} children The items that will be shown as a list in the slider.
+ * @return {object} (
+ *   <Branding modifierClasses={modifierClasses}
+ *             text={text}
+ *   />
+ * )
+ */
+export const Slider = ({ children }) => {
+  // Swiper reference to control scrolling via Buttons
+  const [swiper, setSwiper] = useState(null);
+
+  // Put each child of the component into a separate slide
+  const slides = children.map((child, i) =>
+    <SwiperSlide key={i}>
+      {child}
+    </SwiperSlide>);
+
+  return (
+    <div>
+      {/* Slider itself */}
+      <Swiper
+        spaceBetween={30}
+        freeMode={true}
+        onSwiper={(swiperInstance) => {
+          // Save this Swiper's instance when initialized
+          setSwiper(swiperInstance);
+        }}
+        breakpoints={{
+          // tablet
+          640: {
+            width: TABLET_SCREEN_SIZE,
```

Comment:
Opened your branch and checked this out. Remove the width property, it's not helping with responsive behavior and it will cause issues once we place the Slider inside a layout. The responsive behavior is already handled by the `breakpoints` and the`slidesPerView`